valuesets/ValueSet-UKCore-PracticeSettingCode.xml

```diff
@@ -23,4 +23,445 @@
 			<system value="https://fhir.hl7.org.uk/CodeSystem/UKCore-PracticeSettingCode"/>
 		</include>
 	</compose>
+	<expansion>
+		<identifier value="a2fd6fc7-80a8-46b2-aa43-0ef08bd01373"/>
+		<timestamp value="2023-01-23T14:16:59+00:00"/>
+		<total value="88"/>
```

Comment:
I get 87 codes when I do a line count on this CodeSystem using this table as a reference https://simplifier.net/hl7fhirukcorer4/ukcore-practicesettingcode-duplicate-2, and there are 435 lines in terms of the <contains> elements, which each have 5 lines, 87 * 5 = 437.  Did a sort and a diff and they all match this table.

Reply:
Thats because I'm an idiot, and m,anaged to count the include when I went through the file!

New commit with the right total, and sorted by code as per codesystem